config/unicorn.rb

```diff
@@ -1,3 +1,17 @@
+# Make sure elasticsearch is configured correctly
+require 'rake'
+begin
+  Rake.application.init
+  Rake.application.load_rakefile
+  Rake.application['search:validate_index'].invoke
+rescue => e
+  # Magic exit code expected by forum-supervisor.sh for when
+  # rake search:validate_index fails
+  puts "ElasticSearch configuration validation failed. "\
+       "\"rake search:validate_index\" failed with the following message: #{e.message}"
+  exit(101)
+end
```

Comment:
@maxrothman: Unit tests are not really setup for this.  Using spec_helper, I get a premature error in your file when running `Rake.application.load_rakefile` because it seems some dependencies are already set up.  I already sepnt a while, and I don't think it is worth the effort.